frontend-react/src/typings/api-codegen.ts

```diff
@@ -0,0 +1,150 @@
+type Topic = "FULL_ELR" | "COVID_19" | "MONKEYPOX" | "CSV_TESTS" | "TEST";
+
+interface ReportStreamFilters {
+    jurisdictionalFilter: string[] | undefined;
+    processingModeFilter: string[] | undefined;
+    qualityFilter: string[] | undefined;
+    routingFilter: string[] | undefined;
+    topic: Topic;
+}
+
+type Jurisdiction = "FEDERAL" | "STATE" | "COUNTY";
+
+interface Organization {
+    countyName: string | undefined;
+    description: string;
+    filters: ReportStreamFilters[] | undefined;
+    jurisdiction: Jurisdiction;
+    name: string;
+    stateCode: string | undefined;
+}
+
+interface TemporalAccessor {
```

Comment:
Huhhhh interesting that these Java interfaces are exposed here as empty objects.  I guess it makes sense for proper inheritance, but it just feels odd 😆